SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -92,6 +104,87 @@ void USpatialWorkerConnection::Connect(bool bInitAsClient)
 	}
 }
 
+#pragma optimize("", off)
+void USpatialWorkerConnection::OnLoginTokens(void* UserData, const Worker_Alpha_LoginTokensResponse* LoginTokens)
+{
+	if (LoginTokens->status.code != WORKER_CONNECTION_STATUS_CODE_SUCCESS)
+	{
+		UE_LOG(LogSpatialWorkerConnection, Error, TEXT("Failed to get login token, StatusCode: %d, Error: %s"), LoginTokens->status.code, UTF8_TO_TCHAR(LoginTokens->status.detail));
+		return;
+	}
+
+	if (LoginTokens->login_token_count == 0)
+	{
+		UE_LOG(LogSpatialWorkerConnection, Warning, TEXT("No deployment found to connect to. Did you add the 'dev_login' tag to the deployment you want to connect to?"));
+		return;
+	}
+
+	UE_LOG(LogSpatialWorkerConnection, Log, TEXT("Success: Login Token Count %d"), LoginTokens->login_token_count);
```

Comment:
do you want those log statements that simply print SUCCESS? We usually tend to have as little log as possible when everything works as expected,  but then again leaving it to you and the Unreal GDK team to decide :)

Reply:
@m-samiec what do you suggest?

Reply:
Having a log for one-off events is okay, but `success` is a little thin on explanation. Could you flesh out the log here?